test/NaCl.Core.Benchmarks/Salsa20Benchmark.cs

```diff
@@ -0,0 +1,83 @@
+﻿namespace NaCl.Core.Benchmarks
```

Comment:
We could switch to scoped namespace declaration here since we already using it on the other files =)